readme.md

```diff
@@ -176,6 +176,19 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+#### [Schizophrenia](https://en.wikipedia.org/wiki/Schizophrenia)
+
+>A mental disorder characterized by delusions, hallucinations, or other symptoms of lack of contact with reality.
+
+The term ‘schizophrenia’ was coined in 1910 by the Swiss psychiatrist Paul Eugen Bleuler, and is derived from the Greek words ‘schizo’ (split) and ‘phren’ (mind). The oldest available description of an illness closely resembling schizophrenia can be found in the Ebers papyrus, which dates back to the Egypt of 1550 BC. It is a serious disorder which causes a disconnect between what a person perceives as reality and what truly is reality. This mental gap can be extremely confusing and upsetting, causing intense psychological distress.
+
+[R: nihm.gov](https://www.nimh.nih.gov/health/topics/schizophrenia/index.shtml)
+
+[R: webmd.com](https://www.webmd.com/schizophrenia/mental-health-schizophrenia#1)
```

Comment:
No webmd please.

Reply:
@icyflame I wanted to know the reason behind this. Why no webmd !?